client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -1,6 +1,8 @@
 import { LINK_IN_BIO_FLOW, LINK_IN_BIO_TLD_FLOW } from '@automattic/onboarding';
 import { LaunchpadFlowTaskList, Task } from './types';
 
+export const DOMAIN_UPSELL = 'domain_upsell';
```

Comment:
Just wondering if Is there a specific reason for adding / using the const in this file? We don't do that for any of the other task id, and it doesn't seem like it would be 100% needed for the PR.

Reply:
I agree that it's a bit jarring to see it as the only const that exists. I'll probably update the other strings in the file to match this convention